src/index.js

```diff
@@ -51,12 +53,13 @@ const ping = (options, shouldMute) => {
     // Fires all requests
     .then(response => {
       log(`Loading details of the latest ${options.limit} top stories`);
-
-      return fetchTopStoriesDetails(response, options.limit);
+      return fetchTopStoriesDetails(response);
     })
     // Returns a formatted array with the response request
     .then(response => {
-      return response.map(item => item.body);
+      const listItems = [];
```

Comment:
I've had to normalize the output if you return this `return options.latest ? latest : response;` if it is not latest the app will fires an error, because it the `response` have the whole api response.